src/script/ceph-debug-docker.sh

```diff
@@ -85,7 +85,7 @@ function main {
     T=$(mktemp -d)
     pushd "$T"
     distro="${env/://}"
-    api_url="https://shaman.ceph.com/api/search/?status=ready&project=ceph&flavor=${FLAVOR}&distros=${distro}/x86_64&ref=${branch}&sha1=${sha}"
+    api_url="https://shaman.ceph.com/api/search/?status=ready&project=ceph&flavor=${FLAVOR}&distros=${distro}/$(arch)&ref=${branch}&sha1=${sha}"
```

Comment:
do we set an `arch` var somewhere or do you expect it in the environment?

Since this script is primarily for debugging core files from teuthology, hard coding x86_64 made sense to me.

Reply:
@batrick please note i am replacing `x86_64` with `$(arch)` instead of `${arch}`.

Reply:
ah, my eyes are bad. Thanks!